pyproject.toml

```diff
@@ -72,38 +72,61 @@ build-backend = "poetry.core.masonry.api"
 
 
 [tool.isort]
+py_version = 36
 profile = "black"
 force_single_line = true
-atomic = true
-include_trailing_comma = true
-lines_after_imports = 2
+combine_as_imports = true
 lines_between_types = 1
-use_parentheses = true
+lines_after_imports = 2
 src_paths = ["poetry", "tests"]
-skip_glob = ["*/setup.py"]
-filter_files = true
-known_first_party = "poetry"
+extend_skip = ["setup.py"]
 
 
 [tool.black]
-line-length = 88
+target-version = ['py36']
 include = '\.pyi?$'
```

Comment:
Exclude actually wasn't doing anything here -- modern black has a much better approach that ignores files even when pre-commit passes them directly.